app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<div class="w-100 mt-5">
+<div class="d-flex justify-content-center flex-column">
+<h2 class="display-3 mb-5">Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+
+
+<div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions mt-2">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ ] On submitting the registration form it is not easy to tell if a user has been registered successfully or not. This is because there is no confirmation message and the user is not redirected. Kindly redirect the user and display a success message on the register.
- [ ] Kindly display error messages in case the user tries to submit a form with empty fields.